backend/src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java

```diff
@@ -33,193 +33,175 @@
     private static final Logger LOGGER = Logger.getLogger(EndpointResources.class);
 
     @Inject
-    Mutiny.SessionFactory sessionFactory;
+    EntityManager entityManager;
 
-    @Inject
-    CommonStateSessionFactory commonStateSessionFactory;
```

Comment:
`EndpointResources` is no longer using `CommonStateSessionFactory` in this PR because it made the migration easier.

I wonder if this class still makes sense though because `backend` only uses stateful sessions and `engine` only uses stateless sessions. We can migrate it to Hibernate ORM later alongside with the `engine` module.

Reply:
It makes sense to remove it.